src/library/rhythmbox/rhythmboxfeature.cpp

```diff
@@ -91,13 +91,14 @@ RhythmboxFeature::~RhythmboxFeature() {
     delete m_pRhythmboxPlaylistModel;
 }
 
-BaseSqlTableModel* RhythmboxFeature::getPlaylistModelForPlaylist(const QString& playlist) {
-    BaseExternalPlaylistModel* pModel = new BaseExternalPlaylistModel(
-                                            this, m_pLibrary->trackCollections(),
-                                            "mixxx.db.model.rhythmbox_playlist",
-                                            "rhythmbox_playlists",
-                                            "rhythmbox_playlist_tracks",
-                                            m_trackSource);
+std::unique_ptr<BaseSqlTableModel>
+RhythmboxFeature::createPlaylistModelForPlaylist(const QString& playlist) {
```

Comment:
just to make sure I understand the fix. The issue was basically that an oversight during refactoring resulted in the wrong member function being called?

Reply:
Yes, the getPlaylistModelForPlaylist() is no longer called, instead the default base class implementation is used returning null.